README.md

```diff
@@ -0,0 +1,57 @@
+# ScyllaDB Web Install
+
+Welcome to ScyllaDB Web Install repository!  
+
+ScyllaDB Web Install is a platform-agnostic installation script that allows in easy way to install [ScyllaDB](https://www.scylladb.com/) product on different Linux OSs and on x86_64 / ARM64 machine types.  
+For more information, ScyllaDB Web Install user doc can be found [here](https://opensource.docs.scylladb.com/branch-5.4/getting-started/installation-common/scylla-web-installer.html).
+
+## Usage
+To install the latest official Open Source version of ScyllaDB by Web Installer script, run:
+```shell
+./server
+```
+The script will automatically detect your operating system and will install all the required packages / dependencies.  
```

Comment:
When running only `./server` it will run the latest available OSS release unless provided also product name. I think it worth mention it